collection_requirements.rst

```diff
@@ -167,6 +167,9 @@ We should avoid FQCN / repository names:
 * which are unnecessary long: try to make it compact but clear
 * contain the same words / collocations in ``NAMESPACE`` and ``COLLECTION`` parts, for example ``my_system.my_system``
 
+If a collection is published on Galaxy and is going to be published on Red Hat Automation Hub and vice versa,
+it is desirable that the Galaxy and Automation Hub collection names are identical if possible.
```

Comment:
Suggestion: namespace and collection names

Reply:
Maybe a short explanation on why it is desirable, there is no reference to what problem this tries to solve.

Reply:
How about:
"Having two different names makes it impossible to mention the correct FQCNs in books, tutorials, documents, since there is no single FQCN that always works, and it will cause confusion for users."

Reply:
Looks good to me, thanks for the suggestions!